ding/framework/tests/test_middleware_pace_controller.py

```diff
@@ -0,0 +1,110 @@
+import pytest
+import unittest
+from unittest import mock
+from unittest.mock import patch
+import pathlib as pl
+import os
+import shutil
+from typing import Callable
+
+from ding.framework import Task, Context
+from ding.framework import Parallel
+from ding.framework.middleware import Pace_Controller
+
+
+@pytest.mark.unittest
+class TestPaceControllerModule:
```

Comment:
You should test this middleware in parallel mode with multiprocessing, see `framework/tests/test_parallel.py`